app/src/main/java/com/antosik/benchproject/app/more/view/MoreFragment.kt

```diff
@@ -18,8 +47,134 @@ class MoreFragment : NavigableFragment<MoreFragmentNavigator>() {
         inflater: LayoutInflater,
         container: ViewGroup?,
         savedInstanceState: Bundle?,
-    ) = MoreFragmentBinding.inflate(inflater, container, false).apply {
-        lifecycleOwner = viewLifecycleOwner
-        viewModel = moreViewModel
-    }.root
+    ) = ComposeView(requireContext()).apply {
+        setViewCompositionStrategy(ViewCompositionStrategy.DisposeOnViewTreeLifecycleDestroyed)
```

Comment:
Is it needed, why?

Reply:
I based on this https://www.youtube.com/shorts/dIo4u-FtN30 :P

Reply:
So if you decided to use it in your fragments it would be nice to create some base fragment, probably you don't want to repeat that in every fragment with ComposeView 😄

Reply:
It seems like a good idea, I created an issue for this 😄 #139